ui/lib/kmip/addon/templates/components/edit-form-kmip-role.hbs

```diff
@@ -37,7 +37,7 @@
         <hr />
         <div class="is-flex">
           <div class="kmip-role-allowed-operations">
-            {{#each-in this.model.operationFormFields.firstObject as |groupName fieldsInGroup|}}
+            {{#each-in (get this.model.operationFormFields 0) as |groupName fieldsInGroup|}}
```

Comment:
same comment about 0 being an integer, seems like this doesn't have to be a string? Good to know!